integration_tests/assets/docker-compose.yml

```
@@ -22,3 +22,15 @@ services:
     image: thread-exception
     volumes:
       - ../..:/usr/local/src/xivo-lib-python:ro
+
+  wsgi-patch:
+    image: wsgi-patch
+    volumes:
+      - ../..:/usr/local/src/xivo-lib-python:ro
+    ports:
+      - "8080:8080"
```

```suggestion
      - "8080"
```
We don't want to bind a specific host port, as we don't know if that port is available in an arbitrary host environment(such as CI). Without a `{port}:` prefix, docker compose will assign an arbitrary available port.

---

Updated